llama_stack/providers/remote/inference/together/together.py

```diff
@@ -253,4 +253,9 @@ async def embeddings(
         model_id: str,
         contents: List[InterleavedTextMedia],
     ) -> EmbeddingsResponse:
-        raise NotImplementedError()
+        model = await self.model_store.get_model(model_id)
+        r = self._get_client().embeddings.create(
+            model=model.provider_resource_id, input=contents
```

Comment:
does this actually work with the `InterleavedTextMedia` type as is? that seems impossible to believe actually. What if there's an image in there?

Reply:
most providers accept a str or List[str] for input. I think most of the embedding models are text based and not even support images. I will update this to handle the input based on the supported input types.

Reply:
just realized we have interleaved_text_media_as_str. using that.

Reply:
bedrock actually supports embeddings for image: https://aws.amazon.com/blogs/machine-learning/build-a-reverse-image-search-engine-with-amazon-titan-multimodal-embeddings-in-amazon-bedrock-and-aws-managed-services/#:~:text=To%20convert%20images%20to%20vectors,optimize%20for%20speed%20and%20performance.